dataprofiler/profilers/profiler_options.py

```diff
@@ -866,7 +866,13 @@ def _validate_helper(self, variable_path: str = "OrderOptions") -> list[str]:
 class CategoricalOptions(BaseInspectorOptions):
     """For configuring options Categorical Column."""
 
-    def __init__(self, is_enabled: bool = True, top_k_categories: int = None) -> None:
+    def __init__(
+        self,
+        is_enabled: bool = True,
+        top_k_categories: int = None,
+        max_sample_size_to_check_stop_condition: int = None,
+        stop_condition_unique_value_ratio: float = None,
```

Comment:
The typing should have the `| None`
`max_sample_size_to_check_stop_condition: int | None = None`
same for both.

Reply:
pushed